docs/docs/spec.md

```diff
@@ -426,7 +432,8 @@ If none of the enclosing declarations meet any rule, then the type usage is
 ## Augmented type of a type usage appearing in code {#augmented-type-of-usage}
 
 This section defines how to determine the [augmented types] of most type usages
-in source code or bytecode where JSpecify nullness annotations are [recognized].
+in source code or bytecode where JSpecify nullness annotations are
+[recognized](#recognized-type-use).
```

Comment:
I spent a few minutes on trying to figure out whether we could get Docusaurus to fail for broken links like these. But we already set the settings that look relevant:

https://github.com/jspecify/jspecify/blob/996b10e7786a46a9d31d0110d4c70e95258a28bd/docs/docusaurus.config.js#L29-L30

Maybe it's just technically "not even a link" if the contents of the bracket don't have a corresponding link definition at the end. Oh, well. I guess that we just search the page for `[` (and ignore code samples about arrays :)). Thanks for fixing this one.